.travis.yml

```diff
@@ -13,4 +13,5 @@ cache:
   - "$HOME/.m2"
 install: true
 script:
-  - mvn clean verify -e
+  - chmod +x vsBuild.sh
+  - ./vsBuild.sh

```

Comment:
Could you document somewhere what this is doing? E.g., in vsBuild.sh itself.

Reply:
This is already documented, see https://github.com/tcerny/VariantSync/wiki/Maven

Reply:
Ok great! Could you add this link to `vsBuild.sh`.

Reply:
Since we will move the wiki to Thuem, I dont think linking at the moment is possible

Reply:
Good point!